src/libguac/guacamole/user.h

```diff
@@ -89,6 +89,12 @@ struct guac_user_info {
      */
     int optimal_resolution;
     
+    /**
+     * The human-readable name of the Guacamole user. If the client does not
+     * provide a name then this will be NULL.
+     */
+    const char* username;
```

Comment:
With this being an arbitrary, human-readable value, with no guarantees of uniqueness nor requirements for uniqueness, I think:

* `name` would be a better choice than `username`, which otherwise implies uniqueness.
* We should note that no requirement is imposed on the nature of this value, including whether it is unique, except that it will be the value supplied during the handshake.

Reply:
Okay, I've renamed this from `username` to `name` throughout the code, including during the various handshake operations. Just need to update client-side code, now...